docs/usage.md

```diff
@@ -3,51 +3,82 @@
 
-- Visualize relationship data as per duration. User can select from standard durations provided as part of widget like 7 Days, 1 Day, and 1 Hour. 
+You can select from following options:
 
+- 7 Days
+- 1 Day
+- 1 Hour
+
+    ![](./res/sankey-chart-time-intervals.png)
+<!-- 
 - Configure the widget by selecting source node, target node, relationship node and setting filters to retrieve the required data.
 
-- The links colors will be picked as per picklist items configuration.
+- The links colors will be picked as per picklist items configuration. -->
 
 ## SOC Overview Sankey Widget
 
 1. Edit a *Dashboard*, or a *Report* and select **Add Widget** button.
 
+    ![](./res/sankey-edit-00.png)
+
 2. Select **SOC Overview Sankey** from the list to bring up the **SOC Overview Sankey** widget's customization modal.
 
-3. Specify the title of the visualisation of relationship data as per module in the **Title** field.
+    ![](./res/sankey-edit-01.png)
+
+3. Specify the title of the visualization of relationship data as per module in the **Title** field.
+
+    ![](./res/sankey-edit-02.png)
 
-4. Select the module, whose data to be visualize as per relationship, in the **Resource** field.
+4. Select the module, whose data is to be represented as a Sankey chart as per relationship, in the **Resource** field.
 
-5. Define the filter criteria using which to filter the data retrieved by this widget (Optional).
+    ![](./res/sankey-edit-03.png)
+
+5. Define the filter criteria using which to filter the data retrieved by this widget (Optional), in the **Filter Criteria** field. To know more about using the filter criteria, refer to the [Nested Filter](https://docs.fortinet.com/document/fortisoar/7.4.3/user-guide/207943/dashboards-templates-and-widgets#Nested-Filters) section of the FortiSOAR&trade; user guide.
+
+    ![](./res/sankey-edit-04.png)
 
 6. Select the text field to be grouped for source nodes, in the **Source Nodes** field.
 
-7. Select the picklist field to group data for target nodes, in the **Target Nodes** field.
+    ![](./res/sankey-edit-05.png)
+
+7. Select the field to group data for target nodes, in the **Target Nodes** field.
+
+    ![](./res/sankey-edit-06.png)
 
 8. Select the relationship module, to group data to visualize as per relation, in the **Relationship Resource** field.
 
+    ![](./res/sankey-edit-07.png)
+
 9. Select the picklist field of selected relationship resource to group the data for relationship nodes, in the **Relationship Nodes** field.
 
+    ![](./res/sankey-edit-08.png)
+
 10. Input JSON data to render outer source nodes and linking with modules source, in the **External Source JSON** field.
 
+    ![](./res/sankey-edit-09.png)
+
 ### SOC Overview Sankey Widget - View
 
 The following image displays a **SOC Overview Sankey** widget based on an example where you might want to view the data as per severity of related incidents of alerts group by source and severity:
```

Comment:
We should also render and illustrate one more example here taken from your overview section, say alert --> type --> severity --> escalated/not escalated or you can pick other example as well here